layouts/page/polars-gpu-engine.html

```diff
@@ -250,10 +250,25 @@ <h2 class="accordion-header">
             <div class="accordion-item">
               <h2 class="accordion-header">
                 <button class="accordion-button wrappable-button" type="button" data-bs-toggle="collapse" data-bs-target="#collapse6" aria-expanded="false" aria-controls="collapse6">
+                  What parameters can I pass to the GPUEngine?
```

Comment:
This is in all caps (like "GPUENGINE") when the docs are rendered, I think. Can we add a space?
```suggestion
                  What parameters can I pass to the GPU Engine?
```

Reply:
I'm also OK with leaving as is.